scripts/test/hyriseBenchmarkTPCC_test.py

```diff
@@ -54,8 +56,21 @@ def main():
     close_benchmark(benchmark)
     check_exit_status(benchmark)
 
-    if return_error:
-        sys.exit(1)
+    output_filename_2 = f"{build_dir}/tpcc_output_2.json"
```

Comment:
Consistency: In the other files, you define these at the top.

Reply:
Can you name these files? I think it's always on top of the current benchmark execution.

Reply:
hyriseBenchmarkFileBased_test.py

Reply:
Changed in this file to define the vars where they are needed.